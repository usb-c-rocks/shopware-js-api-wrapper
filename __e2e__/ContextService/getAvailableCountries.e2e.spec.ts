import { getAvailableCountries } from "@shopware-js-api-wrapper";

describe("shopware-6-client - E2E - ContextService - getAvailableCountries", () => {
  it("should test countries response", async () => {
    const result = await getAvailableCountries();
    expect(result).toMatchSnapshot();
  });
});
