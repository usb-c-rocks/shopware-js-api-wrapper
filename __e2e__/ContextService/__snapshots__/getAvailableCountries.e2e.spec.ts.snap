// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shopware-6-client - E2E - ContextService - getAvailableCountries should test countries response 1`] = `
Object {
  "aggregations": Array [],
  "data": Array [
    Object {
      "_uniqueIdentifier": "02f0495069f6465d880164d38337a598",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "02f0495069f6465d880164d38337a598",
      "iso": "AE",
      "iso3": "ARE",
      "name": "Arab Emirates",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Arab Emirates",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "0f2c9e64c8be450bb777623e371d4111",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "0f2c9e64c8be450bb777623e371d4111",
      "iso": "PL",
      "iso3": "POL",
      "name": "Poland",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Poland",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "16836bc3a4b24dfe9f130b2e9f173688",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "16836bc3a4b24dfe9f130b2e9f173688",
      "iso": "IS",
      "iso3": "ISL",
      "name": "Iceland",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Iceland",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "22b8a67b056740b9bd412787e5226c1f",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "22b8a67b056740b9bd412787e5226c1f",
      "iso": "AU",
      "iso3": "AUS",
      "name": "Australia",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Australia",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "2cc8f9dbbe0a41458d35316f740e61d1",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "2cc8f9dbbe0a41458d35316f740e61d1",
      "iso": "PT",
      "iso3": "PRT",
      "name": "Portugal",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Portugal",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "3703cd75fabb4799a358dcb59cd97192",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "3703cd75fabb4799a358dcb59cd97192",
      "iso": "JP",
      "iso3": "JPN",
      "name": "Japan",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Japan",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "3c9de310f5774670b564d405d3fad662",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "3c9de310f5774670b564d405d3fad662",
      "iso": "LU",
      "iso3": "LUX",
      "name": "Luxembourg",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Luxembourg",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "414f148670a54145a3892d5328b79b8b",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "414f148670a54145a3892d5328b79b8b",
      "iso": "IE",
      "iso3": "IRL",
      "name": "Ireland",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Ireland",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "455a0f4f6254409ca80f8a2cb78bfb31",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:19+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "455a0f4f6254409ca80f8a2cb78bfb31",
      "iso": "GB",
      "iso3": "GBR",
      "name": "United Kingdom",
      "position": 5,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "United Kingdom",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "542220692da649188a31f0d13002207c",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:19+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "542220692da649188a31f0d13002207c",
      "iso": "DE",
      "iso3": "DEU",
      "name": "Germany",
      "position": 1,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Germany",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "57b642edd0464db3807f8aa631248387",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "57b642edd0464db3807f8aa631248387",
      "iso": "RO",
      "iso3": "ROU",
      "name": "Romania",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Romania",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "6c77663043dc472f9c077c5d10dd35a8",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "6c77663043dc472f9c077c5d10dd35a8",
      "iso": "NO",
      "iso3": "NOR",
      "name": "Norway",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Norway",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "6e1ae1b89f754f60b07a90fc5fe7c4fa",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "6e1ae1b89f754f60b07a90fc5fe7c4fa",
      "iso": "BE",
      "iso3": "BEL",
      "name": "Belgium",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Belgium",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "75f4d2b6cf09423ca368e8541789a387",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "75f4d2b6cf09423ca368e8541789a387",
      "iso": "BR",
      "iso3": "BRA",
      "name": "Brazil",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Brazil",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "7e12d5be80d74e21907641e2a7128afd",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "7e12d5be80d74e21907641e2a7128afd",
      "iso": "IL",
      "iso3": "ISR",
      "name": "Israel",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Israel",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "7f3157db976347b68a32d150db8534a6",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "7f3157db976347b68a32d150db8534a6",
      "iso": "LI",
      "iso3": "LIE",
      "name": "Liechtenstein",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Liechtenstein",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "84d12c5098114b8fa568ed156f9ea8f9",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "84d12c5098114b8fa568ed156f9ea8f9",
      "iso": "HU",
      "iso3": "HUN",
      "name": "Hungary",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Hungary",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "87865c9bf03249a18be3d121f891d458",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "87865c9bf03249a18be3d121f891d458",
      "iso": "CA",
      "iso3": "CAN",
      "name": "Canada",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Canada",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "8efc7f55cce74678a51b58120dbdb57b",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "8efc7f55cce74678a51b58120dbdb57b",
      "iso": "NA",
      "iso3": "NAM",
      "name": "Namibia",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Namibia",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "96b73d0ed38643e1acfef0bc67a6568f",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "96b73d0ed38643e1acfef0bc67a6568f",
      "iso": "SK",
      "iso3": "SVK",
      "name": "Slovenia",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Slovenia",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "99568fde28d94706a911b6df11822eee",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "99568fde28d94706a911b6df11822eee",
      "iso": "FR",
      "iso3": "FRA",
      "name": "France",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "France",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "9daf52ef933e4657bce272ce23789297",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "9daf52ef933e4657bce272ce23789297",
      "iso": "CH",
      "iso3": "CHE",
      "name": "Switzerland",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": true,
      "translated": Object {
        "customFields": Array [],
        "name": "Switzerland",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "9eadb61c416b486395e3d8392ee84e06",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "9eadb61c416b486395e3d8392ee84e06",
      "iso": "CZ",
      "iso3": "CZE",
      "name": "Czech Republic",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Czech Republic",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "9feeaf77dbea4b99b3b459e7f0b40a57",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "9feeaf77dbea4b99b3b459e7f0b40a57",
      "iso": "IT",
      "iso3": "ITA",
      "name": "Italy",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Italy",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "a8a7344301b24145be9b9ce20b49d67e",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "a8a7344301b24145be9b9ce20b49d67e",
      "iso": "TR",
      "iso3": "TUR",
      "name": "Turkey",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Turkey",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "ac44450979204978b69490db7d6d754b",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "ac44450979204978b69490db7d6d754b",
      "iso": "FI",
      "iso3": "FIN",
      "name": "Finland",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Finland",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "b4cb00937e6842cbb891f6c574622183",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:19+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "b4cb00937e6842cbb891f6c574622183",
      "iso": "GR",
      "iso3": "GRC",
      "name": "Greece",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Greece",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "bcc07b3cd73d443db6290e6b9dcae18b",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "bcc07b3cd73d443db6290e6b9dcae18b",
      "iso": "NL",
      "iso3": "NLD",
      "name": "Netherlands",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Netherlands",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "becac85a044a4e7883d0f22e31f430eb",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:21+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "becac85a044a4e7883d0f22e31f430eb",
      "iso": "DK",
      "iso3": "DNK",
      "name": "Denmark",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Denmark",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "c113485bc6fa45be80ddfb20f2ac7136",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "c113485bc6fa45be80ddfb20f2ac7136",
      "iso": "US",
      "iso3": "USA",
      "name": "USA",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "USA",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "c74710c2a2d04b1eba6dd3340dc809dd",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "c74710c2a2d04b1eba6dd3340dc809dd",
      "iso": "ES",
      "iso3": "ESP",
      "name": "Spain",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Spain",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "e090203e690f4441a8d37b58deb7497c",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "e090203e690f4441a8d37b58deb7497c",
      "iso": "SE",
      "iso3": "SWE",
      "name": "Sweden",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Sweden",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
    Object {
      "_uniqueIdentifier": "e1fc954d5eca4051883f5750af01b5f9",
      "active": true,
      "apiAlias": "country",
      "createdAt": "2020-04-28T13:55:20+00:00",
      "customFields": null,
      "displayStateInRegistration": false,
      "extensions": Object {
        "foreignKeys": Object {
          "extensions": Array [],
        },
      },
      "forceStateInRegistration": false,
      "id": "e1fc954d5eca4051883f5750af01b5f9",
      "iso": "AT",
      "iso3": "AUT",
      "name": "Austria",
      "position": 10,
      "shippingAvailable": true,
      "states": null,
      "taxFree": false,
      "translated": Object {
        "customFields": Array [],
        "name": "Austria",
      },
      "translations": null,
      "updatedAt": null,
      "versionId": null,
    },
  ],
  "total": 33,
}
`;
